commit/metrics/prom.go

```diff
@@ -153,6 +176,17 @@ func (p *PromReporter) TrackMerkleOutcome(outcome merkleroot.Outcome, state stri
 	}
 }
 
+func (p *PromReporter) TrackRmnReport(latency float64, destChainSelector uint64, success bool) {
+	destChainSelectorStr := strconv.FormatUint(destChainSelector, 10)
+	successStr := strconv.FormatBool(success)
+	p.merkleProcessorRmnReportHistogram.WithLabelValues(destChainSelectorStr, successStr).Observe(latency)
+}
+
+func (p *PromReporter) TrackRmnRequest(method string, latency float64, nodeID uint64, err string) {
```

Comment:
I wondering if this can cause an issue with the label cardinality if we label with the err. Unless the errors are bounded by used a know set of error code we risk have an unbounded label here?

Reply:
Currently the set of errors is bounded, they're all statically defined (as opposed to created from converting arbitrary errors to string), so there's little risk of cardinality issues.